docs/software/navigation_packages.mdx

```diff
@@ -13,7 +13,7 @@ sales@clearpathrobotics.com for more information on these software packages.
 
 - [Product Overview](https://clearpathrobotics.com/outdoor-autonomy-software/)
 - User Manual
-  - [Latest (0.7.0)](/docs_outdoornav_user_manual)
+  - [Latest (0.8.0)](/docs_outdoornav_user_manual)
   <!-- - [0.7.0](/docs_outdoornav_user_manual/0.7.0) -->
```

Comment:
Do you need to uncomment this 0.7.0 link? It assume we still want access to the old documentation.

Reply:
Agreed. We still want a link to the 0.7.0 documentation.

Reply:
@dobrienCP the repository is missing a versioned folder for 0.8.0. 
Take a look at these instructions on the README for adding a new version of OutdoorNav:
https://github.com/clearpathrobotics/cpr-documentation#workflow-for-publishing-outdoornav-user-manual-updates

Reply:
The link has been restored and a new version has been created. There is a slight problem with this process I noticed in that the images that are used are not preserved between versions. I will make a ticket to investigate this further.

Reply:
For images, the best way to handle this in the short term is likely to have two copies of the image, with the version in the name, so that we can select the appropriate image for the new version being updated.